content/guide/07-element-directives.md

```diff
@@ -130,29 +130,6 @@ When the user clicks a button, the component will fire a `select` event, where t
 </script>
 ```
 
-```html
-<!--{ filename: 'CategoryChooser.html', hidden: true }-->
-<p>Select a category:</p>
-
-{#each categories as category}
-	<button on:click="fire('select', { category })">select {category}</button>
-{/each}
-
-<script>
-	export default {
-		data() {
-			return {
-				categories: [
-					'animal',
-					'vegetable',
-					'mineral'
-				]
-			}
-		}
-	};
-</script>
-```
-
```

Comment:
I think we should leave this section here. It is the file that accompanies the "Component events" example above.